1. Calculate the probability of flipping a balanced coin four times and getting each pattern: HTTH, HHHH and TTHH.

2. If a list of people has 24 women and 21 men, then the probability of choosing a man from the list is 21/45. What is the probability of not choosing a man?

3. The probability that Bernice will travel by plane sometime in the next year is 10%. The probability of a plane crash at any time is .005%. What is the probability that Bernice will be in a plane crash sometime in the next year?

4. A data scientist wants to study the behavior of users on the company website. Each time a user clicks on a link on the website, there is a 5% chance that the user will be asked to complete a short survey about their behavior on the website. The data scientist uses the survey data to conclude that, on average, users spend 15 minutes surfing the company website before moving on to other things. What is wrong with this conclusion?

In [1]:
print(.5 * .5 * .5 * .5)

0.0625


1 The probability of reaching any pattern is the same 6.25%. 

In [2]:
print(1/16)

0.0625


1 This is another way to solve question one. There are 16 possible patterns which means a 1 in 16 chance to get any pattern.

In [3]:
print(24/45)

0.5333333333333333


2 The chance you will not select a man is 53% or 24/45.

In [4]:
print(.10 * .005)

0.0005


3 The probability that these two events will occur is .05% or 5 in 1000.

4 People who surf longer are likely to click more links, increasing the odds of getting a survey. People who don't get prompted for a survey are not in the sample. 